website/src/components/CommunityBanner.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+function Banner(): JSX.Element {
+  return (
+    <div className="w-full flex flex-row justify-center items-center py-2 bg-gradient-to-r from-sky-700 to-purple-700 overflow-hidden px-5">
+      <div className="mx-3 relative text-white font-semibold">
+        <img className="inline h-12 align-middle mr-1" alt="Large box" src="img/banner/large_box.png" />
+        <img className="inline h-6 align-bottom mr-10" alt="Small box" src="img/banner/small_box.png" />
+        Join us for the first Podman Desktop Community Meeting on March 27th, 2025, from 9:00-10:00 AM EST! Check out
+        the agenda and RSVP
+        <a
+          href="https://github.com/podman-desktop/community/issues/3"
+          target="_blank"
```

Comment:
added _blank so it opens a new link / new tab, WDYT of the behaviour / it's okay @benoitf